building-srpm-in-copr/README.md

```diff
@@ -90,3 +85,27 @@ copr_client.build_proxy.create_from_custom(ownername="lbarczio", projectname="og
                            script_builddeps=["git", "packit", "python3-wheel", "python3-pip", "python3-setuptools",
                             "python3-setuptools_scm", "python3-setuptools_scm_git_archive"])
 ```
+
+### Plan how to build SRPMs in Copr in Packit Service
+
+- create CLI command or only Packit API method for preparing sources (would mostly use existing functionality, described above)
+- implement creation of the script (Python/bash - Python would be easier for handling with configs)
+  - script will be created dynamically each time
```

Comment:
I touched this topic before ... but if the SRPM build method will have clear inputs (and outputs, but that's just srpm I bet), it would be relatively easy to implement a new "Packit" method in Copr ([aside](https://pagure.io/copr/copr/blob/main/f/rpmbuild/copr_rpmbuild/providers)  from the custom method).  It's up to you what you prefer, and where you want to start/iterate to.